gulpfile.babel.js

```diff
@@ -102,6 +103,14 @@ gulp.task('assets', function() {
 });
 /* End of building assets */
 
+/* Building html */
+gulp.task('default', function () {
```

Comment:
`default` is the name of default task for gulp and has been already defined above. Change it into `html` or something like this.